docs/core/diagnostics/built-in-metrics-runtime.md

```diff
@@ -0,0 +1,242 @@
+
+## `System.Runtime`
+
+The `System.Runtime` Meter reports measurements from the GC, JIT, AssemblyLoader, Threadpool, and exception handling portions of the .NET runtime as well as some CPU and memory metrics from the OS. These metrics are available automatically for all .NET apps.
+
+##### Metric: `dotnet.process.cpu.time`
+
+| Name | Instrument Type | Unit (UCUM) | Description |
+| ---- | --------------- | ----------- | ----------- |
+| `dotnet.process.cpu.time` | Counter | `s` | CPU time used by the process. |
+
+| Attribute | Type | Description | Examples | Presence |
+|---|---|---|---|---|
+| `cpu.mode` | string | The mode of the CPU. | `user`; `system` | Always |
+
+This metric reports the same values as accessing the processor time properties on <xref:System.Diagnostics.Process?displayProperty=nameWithType> for the current process. The `system` mode corresponds to <xref:System.Diagnostics.Process.PrivilegedProcessorTime> and `user` mode corresponds to <xref:System.Diagnostics.Process.UserProcessorTime>
+
+Available starting in: .NET 9.0.
+
+##### Metric: `dotnet.process.memory.working_set`
+
+| Name | Instrument Type | Unit (UCUM) | Description |
+| ---- | --------------- | ----------- | ----------- |
+| `dotnet.process.memory.working_set` | UpDownCounter | `By` | The number of bytes of physical memory mapped to the process context. |
+
+This metric reports the same values as calling <xref:System.Environment.WorkingSet?displayProperty=nameWithType> property.
+
+Available starting in: .NET 9.0.
+
+##### Metric: `dotnet.gc.collections`
+
+| Name | Instrument Type | Unit (UCUM) | Description |
+| ---- | --------------- | ----------- | ----------- |
+| `dotnet.gc.collections` | Counter | `{collection}` | The number of garbage collections that have occurred since the process has started. |
+
+| Attribute | Type | Description | Examples | Presence |
+|---|---|---|---|---|
+| `dotnet.gc.heap.generation` | string | Name of the maximum managed heap generation being collected. | `gen0`; `gen1`; `gen2` | Always |
+
+The .NET GC is a generational garbage collector. Each time the garbage collector runs it uses heuristics to select a maximum generation and then collects objects in all generations up to the selected maximum. For example a `gen1` collection collects all objects in generations 0 and 1. A `gen2` collection collects all objects in generations 0, 1, and 2. For more information about the .NET GC and generational garbage collection see the [.NET Garbage collection guide](/standard/garbage-collection/fundamentals.md#generations).
+
+Available starting in: .NET 9.0.
+
+##### Metric: `dotnet.gc.heap.total_allocated`
+
+| Name | Instrument Type | Unit (UCUM) | Description |
+| ---- | --------------- | ----------- | ----------- |
+| `dotnet.gc.heap.total_allocated` | Counter | `By` | The *approximate* number of bytes allocated on the managed GC heap since the process has started. The returned value does not include any native allocations. |
```

Comment:
```suggestion
| `dotnet.gc.heap.total_allocated` | Counter | `By` | The *approximate* number of bytes allocated on the managed GC heap since the process started. The returned value does not include any native allocations. |
```